util/testutil/integration/oci.go

```diff
@@ -205,6 +213,21 @@ func runBuildkitd(args []string, logs map[string]*bytes.Buffer, uid, gid int) (a
 		return "", nil, err
 	}
 
+	deferF.append(func() error {
+		f, err := os.Open("/proc/self/mountinfo")
+		if err != nil {
+			return errors.Wrap(err, "failed to open mountinfo")
+		}
+		defer f.Close()
+		s := bufio.NewScanner(f)
+		for s.Scan() {
+			if strings.Contains(s.Text(), tmpdir) {
+				return errors.Errorf("leaked mountpoint for %s", tmpdir)
+			}
+		}
+		return nil
```

Comment:
return s.Err()

Reply:
Though I don't see where this error is used. All returns from `deferF.F()` are ignored

Reply:
Ok so only some errors from `deferF.F()()` are ignored, but I now see the assert on the cleanup function after run.go's call to New()